client/src/containers/Home/Home.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import Carousel from "../../components/UI/Carousel/Carousel";
+import { connect } from "react-redux";
+import axios from "axios";
+import Category from "../../components/UI/Category/Category";
+
+function Home(props) {
+  const [banners, setBanners] = React.useState([]);
+  const [categories, setCategories] = React.useState([]);
+
+  React.useEffect(() => {
+    axios
+      .get("http://localhost:5001/banners")
+      .then((res) => setBanners(res.data))
+      .catch((err) => console.log(err));
```

Comment:
keep your component free from axios. Use an external file to fetch data. Import that file in this component and use its return value containing api data

Reply:
Done